docs/settings.rst

```diff
@@ -63,3 +63,17 @@ CRAWLERA_BACKOFF_MAX
 Default: ``180``
 
 Max value for exponential backoff as showed in the formula above.
+
+CRAWLERA_FORCE_ENABLE_ON_HTTP_CODES
+------------------------------------
+
+Default: ``[]``
+
+List of http resṕmse status codes that when, ``scrapy-crawlera`` gets a response with one of them and is disabled, it should retry the request forcing the middleware to use crawlera (just for that request).
+
+CRAWLERA_FORCE_ENABLE_FOR_ALL_REQUESTS
+---------------------------------------
+
+Default: ``False``
+
+Set this to ``True`` when you want that, after a response with a code set on ``CRAWLERA_FORCE_ENABLE_ON_HTTP_CODES``, ``scrapy-crawlera`` be enabled for all subsequent requests. 
```

Comment:
How about limiting it to requests for the domain/netloc that triggered the activation?

Reply:
Might make sense, changes to the middleware would need to be deeper tho... let me think a bit on this.

Reply:
It's surely a much better fit for the broad crawl case.